test/scons-time/run/option/quiet.py

```diff
@@ -64,20 +64,7 @@
                 'foo-000-2.log',
                 'foo-000-2.prof')
 
-scons_py = test.workpath('src/script/scons.py')
-
-src_engine = test.workpath('src/engine')
-
-test.run(arguments = 'run -q foo.tar.gz', stdout = expect)
-
-test.must_exist('foo-001-0.log',
-                'foo-001-0.prof',
-                'foo-001-1.log',
-                'foo-001-1.prof',
-                'foo-001-2.log',
-                'foo-001-2.prof')
-
-test.run(arguments = 'run --quiet foo.tar.gz', stdout = expect)
+test.run(arguments='run --quiet foo.tar.gz', stdout=expect)
```

Comment:
Why remove the rest of test?

Reply:
Blindness. It looks to me like linkes 67 and 69 were no-ops (redefine without ever using for anything), then running the same test with the same test expectation. Except I now see it's not the same expectation, `-001-` vs `-000-`.